spec/features/make_peeps_spec.rb

```diff
@@ -0,0 +1,8 @@
+# feature 'make peeps' do
+#   scenario 'user wants to make a peep' do
+#     visit ('/peeps')
+#     fill_in('make_peep', with: 'Hi Chitter!')
+#     click_button('Submit')
+#     expect(page).to have_content('first peep')
+#   end
+# end
```

Comment:
same here - do you need to keep the commented test?

FYI - you can also skip tests, so that they're pending, instead of commenting them:
```
feature 'make peeps' do
  xscenario 'user wants to make a peep' do
    visit ('/peeps')
    fill_in('make_peep', with: 'Hi Chitter!')
    click_button('Submit')
    expect(page).to have_content('first peep')
  end
end 
```

The x infront of scenario skips the test. works with `it` blocks too.